packages/apps/fortune/exchange-oracle/server/src/modules/job/job.service.spec.ts

```diff
@@ -325,6 +327,25 @@ describe('JobService', () => {
       expect(assignment.status).toBe(AssignmentStatus.VALIDATION);
     });
 
+    it('should fail if assignment status is not ACTIVE', async () => {
+      const assignment = {
+        jobId: 1,
+        workerAddress: workerAddress,
+        status: AssignmentStatus.CANCELED,
+      };
+
+      jest
+        .spyOn(assignmentRepository, 'findOneByEscrowAndWorker')
+        .mockResolvedValue(assignment as AssignmentEntity);
+
+      storageService.downloadJobSolutions = jest.fn().mockResolvedValueOnce([]);
```

Comment:
```suggestion
```
You don't need to mock `storageService` as your exception is going to be thrown before that call